client/test/query/time.js

```diff
@@ -1,13 +1,13 @@
 import { _do as tap } from 'rxjs/operator/do'
 import { toArray } from 'rxjs/operator/toArray'
 
-import { assertCompletes, compareWithoutVersion } from './utils'
+import { assertCompletes, compareWithoutVersion } from '../utils'
 
-const timesSuite = global.timesSuite = getData => () => {
+export default function() {
   let data
 
-  before(() => {
-    data = getData()
+  before(function() {
+    data = this.hz_data
```

Comment:
If we were not using arrow functions (which capture lexical scope) in all of our `it` assertions, we could reference `this.hz_data` and/or `this.horizon` in all our tests directly. This is unfortunate. Down the road it may be worth avoiding arrow functions in the tests so we can more easily manage their context and further avoid referencing things out of scope.